### PR TITLE
Completed react-hot-button-component exercise.

### PR DESCRIPTION
**Pull request by saraysc**
hot-button

### CODE REVIEW

File: react-hot-button-component/src/hot-button.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  } else if (numberClicks >= 9 && numberClicks < 12) {
+    return 'coral';
+  } else if (numberClicks >= 12 && numberClicks < 15) {
+    return 'orange';
+  } else if (numberClicks >= 15 && numberClicks < 18) {
+    return 'yellow';
+  } else if (numberClicks >= 18) {
+    return 'white-background';
+  }
+}
```
Comment (Zhiwutian) [inline react-hot-button-component/src/hot-button.jsx:17]: great idea using a function to get the class name. I would recommend making this a method in the HotButton class, you want all the functionality self contained and built into the component.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Great work! Just make buttonColour a method!

**Comment by saraysc** [comment]
hot-button

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**